### ISSUE TITLE
The extension crashes in Chrome Canary (Chrome 119)

### ISSUE THREAD

**Issue by bramus** [bramus/scroll-driven-animations-debugger-extension#29]
When using the extension in current Chrome Canary _(tested in Chrome 119.0.6005.0)_, the extension crashes when inspecting an element that has a Scroll-Driven Animation attached.

<img width="1567" alt="image" src="https://github.com/bramus/scroll-driven-animations-debugger-extension/assets/213073/201feb93-4483-4e5b-a2f7-a09985575af9">

In the console, it reads this:

```
Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.
```

**Comment by bramus** [comment]
It took some time to investigate, but the reported error in the console is unrelated to the crash _(See [crbug/1340647](https://bugs.chromium.org/p/chromium/issues/detail?id=1340647))_. The connections between the DevTools pane and Content Script work fine, and don’t catch any errors at all.

Turns out it is a GPU crash in Chrome itself. When updating the `currentTime` of the cloned animations in the extension, the extension crashes when doing that too fast after each other.

Inspecting the crash data [crash/d4adab2203e014bd](http://crash/d4adab2203e014bd) _(Google internal only link)_, the CL that caused this change most likely is https://chromium-review.googlesource.com/c/chromium/src/+/4850750. Thankfully a revert CL https://chromium-review.googlesource.com/c/chromium/src/+/4862648 got merged today, so it should be fixed in tomorrow’s Canary build. Fingers crossed … 🤞

**Comment by bramus** [comment]
Verified fixed in Chrome 119.0.6008.0. Closing and updating labels.
